ci-operator/templates/cluster-launch-e2e-upgrade.yaml

```diff
@@ -0,0 +1,333 @@
+kind: Template
+apiVersion: template.openshift.io/v1
+
+parameters:
+- name: JOB_NAME_SAFE
+  required: true
+- name: JOB_NAME_HASH
+  required: true
+- name: NAMESPACE
+  required: true
+- name: IMAGE_FORMAT
+  required: true
+- name: IMAGE_PREVIOUS_FORMAT
```

Comment:
Let's call these `PREVIOUS_*`, as the `IMAGE_*` and `LOCAL_IMAGE_*` vars are internal to `ci-operator`

Reply:
Done. So ci-operator would use any env var to set template params? The docs are a bit vague, I thought it would only use `IMAGE_*` and `TEST_*`?